src/v2/components/design/modal/RegistrationModal.tsx

```diff
@@ -0,0 +1,9 @@
+import styled from 'styled-components';
+
+export type RegistrationModalProps = {
+	children?: JSX.Element;
+};
+export const RegistrationModal = ({ children }: RegistrationModalProps): JSX.Element => {
```

Comment:
Let's just call this modal component
```suggestion
export const Modal = ({ children }: ModalProps): JSX.Element => {
```